todo_app/tests/test_trello_calls.py

```diff
@@ -0,0 +1,51 @@
+import os
+import pytest
+import requests
+from dotenv import load_dotenv, find_dotenv
+from todo_app import app
+
+
+class StubResponse():
+    def __init__(self, fake_response_data):
+        self.fake_response_data = fake_response_data
+
+    def json(self):
+        return self.fake_response_data
+
+
+def stub(request_string, params={}):
```

Comment:
there is some python jank around mutable objects as default values. essentially if this dictionary is ever changed (ie adding a key) this will persist through ALL future methods of the call that use the default param (it doesnt reset). god bless python.
not overly important here because params is unused but generally shouldn't be done